scripts/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  geoviz_test_db:
+    image: postgis/postgis:latest
+    container_name: geoviz_test_db
+    volumes:
+      - ./pgdata:/var/lib/postgresql/data/pgdata
+    ports:
+      - "5400:5432"
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
+      - POSTGRES_PASSWORD=postgres
```

Comment:
Should this file be combined with root docker-compose.yml, or do you think we should have a separate docker-compose.yml for tests? Perhaps put this on root level or create a /test directory for this?